### DISCUSSION TITLE
Use tailwind in an external repository with purge

### DISCUSSION THREAD

**Discussion by kurt-hardy** [tailwindlabs/tailwindcss#5084] [Help]
Hi

I have about 7 different repos which all have there own tailwind setup (EG 7 different repos with 7 different versions of the same button)

I have thus created a private repo to remove this duplication but I'm having issues with JIT and purging. If I remove the purge, it works. Is there a way I can tell tailwind to include the classes from my external repo?

eg ` <Button colour="brand" size={ButtonSize.sm} className="my-8">Sign in</Button>`

Thanks

**Comment by simonswiss** [comment]
Is adding a path to the `node_modules` folder in question not working?

If your packages has a `@` namespace, you might need to escape that:

```diff
- purge: ['./node_modules/@my-package/components/dist/*.js']]
+ purge: ['./node_modules/\\@my-package/components/dist/*.js']]
```

**Reply by kurt-hardy** [reply]
I only had my `src` folder in the purge. I didnt realise that I needed my package!

Thanks for your help @simonswiss 👍

**Reply by simonswiss** [reply]
No problem, glad that helped 👍

**Reply by kurt-hardy** [reply]
Hi @simonswiss 

I thought this was working but its not 100% right.

My button has these classes
`inline-flex justify-center items-center border font-medium rounded focus:outline-none focus:ring-2 focus:ring-opacity-30 transition text-base px-4 py-2 cursor-not-allowed border-transparent text-gray-200 bg-primary-700 shadow-sm rounded mr-4`

But only the classes I have used in the site are applied.  
EG My page uses mr-4, my button uses mr-4 and py-2 but the button only has the mr-4 class applied. 

I have this in my purge content:     '.node-modules/\\@company/react-tailwind/lib/**/*.{ts,tsx}'

Any ideas?
Thanks
